tests/testthat/test-relational.R

```diff
@@ -76,6 +76,13 @@ test_that("we can create various expressions and don't crash", {
   expect_true(TRUE)
 })
 
+test_that("we can create a constant NA with LOGICAL type", {
+  rel1 <- rel_from_df(con, data.frame(x = 42))
+  exprs <- list(
+    y = expr_constant(NA, TRUE)
+  )
+  expect_equal(rel_to_altrep(rel_project(rel1, exprs))[, ], NA)
```

Comment:
Is there a more straightforward way to convert a constant to a SEXP rather that having to round trip through a data frame ?

Reply:
For our use case, this is best. You could add a wrapper function in this package?

Reply:
A function that turns a constant into an R value ?